assets/stylesheets/modules/_diagram.scss

```diff
@@ -0,0 +1,71 @@
+//DIAGRAM SECTION
+
+.diagram {
+  padding-top: 25px;
+  position: relative;
+}
+
+.diagram-detail {
+  padding-top: 65px;
+
+  .diagram-detail__card {
```

Comment:
This (and `diagram-detail__card__content` probably don't need to be nested to do their job. Less nesting is always better.